### PR TITLE
Update tabs

### LINKED ISSUE #416: Update tabs
https://material-ui.com/components/tabs/

### PR DESCRIPTION
**Pull request by dwkaminsky**
## Description
Updated Tabs to latest `material-ui`

## Motivation and Context
Closes #416 

## Screenshots (if appropriate):
captured (7)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,36 +6,39 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+    - Drawer
+    - DropDownMenu
     - FontIcon
     - Paper
-    - Card
+    - RadioButtonGroup
     - Snackbar
-    - Divider
-    - CircularProgress
+    - Toggle
```
Comment (coralvanda) [inline CHANGELOG.md]: Missing `Tab`

File: src/lib/components/Tabs.react.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      this.props.fireEvent({event: 'change'});
+    }
+  };
+
+  render() {
+    const { id, children, tabPropsArray, value, classes} = this.props;
+    this.handleChange = this.handleChange.bind(this)
+
+    return (
+      <div id={id}>
```
Comment (coralvanda) [inline src/lib/components/Tabs.react.js:120]: Need to add the `className` prop

File: src/lib/components/Tabs.react.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        <MuiThemeProvider muiTheme={getMuiTheme(lightBaseTheme)}>
+          <MuiTabs
+            onChange={this.handleChange}
+            value={value}
```
Comment (coralvanda) [inline src/lib/components/Tabs.react.js:124]: Should this be `this.state.value`?

**Comment by dwkaminsky** [comment]
@coralvanda I'm going to merge these one at a time, this one is ready now

File: src/lib/components/Tabs.react.js (hunk 3 above)
Comment (dwkaminsky) [reply src/lib/components/Tabs.react.js:124]: I don't think so, it appears to work with two types of callbacks (demonstrated in the gif)

**Pull request merged**